spec/rails_helper.rb

```diff
@@ -67,3 +69,12 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+
+Capybara.configure do |config|
```

Comment:
can you move this into `spec/support/capybara.rb` - have been trying to keep config and customisation for different components in separate files, for organisational reasons